src/backend/curses/pan.rs

```diff
@@ -0,0 +1,218 @@
+                pancurses::Input::KeyOptions => Event::Refresh,
+                pancurses::Input::KeyPrevious => Event::Refresh,
+                pancurses::Input::KeyRedo => Event::Refresh,
+                pancurses::Input::KeyReference => Event::Refresh,
+                pancurses::Input::KeyRefresh => Event::Refresh,
+                pancurses::Input::KeyReplace => Event::Refresh,
+                pancurses::Input::KeyRestart => Event::Refresh,
+                pancurses::Input::KeyResume => Event::Refresh,
+                pancurses::Input::KeySave => Event::Refresh,
+                pancurses::Input::KeySBeg => Event::Refresh,
+                pancurses::Input::KeySCancel => Event::Refresh,
+                pancurses::Input::KeySCommand => Event::Refresh,
+                pancurses::Input::KeySCopy => Event::Refresh,
+                pancurses::Input::KeySCreate => Event::Refresh,
+                pancurses::Input::KeySDC => Event::Refresh,
+                pancurses::Input::KeySDL => Event::Refresh,
+                pancurses::Input::KeySelect => Event::Refresh,
+                pancurses::Input::KeySEnd => Event::Refresh,
+                pancurses::Input::KeySEOL => Event::Refresh,
+                pancurses::Input::KeySExit => Event::Refresh,
+                pancurses::Input::KeySFind => Event::Refresh,
+                pancurses::Input::KeySHome => Event::Refresh,
+                pancurses::Input::KeySIC => Event::Refresh,
+                pancurses::Input::KeySLeft => Event::Refresh,
+                pancurses::Input::KeySMessage => Event::Refresh,
+                pancurses::Input::KeySMove => Event::Refresh,
+                pancurses::Input::KeySNext => Event::Refresh,
+                pancurses::Input::KeySOptions => Event::Refresh,
+                pancurses::Input::KeySPrevious => Event::Refresh,
+                pancurses::Input::KeySPrint => Event::Refresh,
+                pancurses::Input::KeySRedo => Event::Refresh,
+                pancurses::Input::KeySReplace => Event::Refresh,
+                pancurses::Input::KeySRight => Event::Refresh,
+                pancurses::Input::KeySResume => Event::Refresh,
+                pancurses::Input::KeySSave => Event::Refresh,
+                pancurses::Input::KeySSuspend => Event::Refresh,
+                pancurses::Input::KeySUndo => Event::Refresh,
+                pancurses::Input::KeySuspend => Event::Refresh,
+                pancurses::Input::KeyUndo => Event::Refresh,
+                pancurses::Input::KeyResize => Event::Refresh,
+                pancurses::Input::KeyEvent => Event::Refresh,
+                // TODO: mouse support
+                pancurses::Input::KeyMouse => Event::Refresh,
+                // TODO: not sure what those are
+                pancurses::Input::KeyA1 => Event::Refresh,
+                pancurses::Input::KeyA3 => Event::Refresh,
+                pancurses::Input::KeyB2 => Event::Refresh,
```

Comment:
`KeyB2` seems to be `Key::NumpadCenter`.
The other keys were probably the rest of the numpad before it got converted to Home/Pgup etc.